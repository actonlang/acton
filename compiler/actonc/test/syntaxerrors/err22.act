# Invalid comprehension syntax
def bad_comp():
    return [x for x in]