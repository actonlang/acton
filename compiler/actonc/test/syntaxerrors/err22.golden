Building file test/syntaxerrors/err22.act using temporary scratch directory
[error Parse error]: unexpected "]<newline>"
                     expecting relational expression

     +--> test/syntaxerrors/err22.act@3:23-3:24
     |
   3 |     return [x for x in]
     :                       ^
     :                       `- unexpected "]<newline>"
     :                          expecting relational expression
     :                          
-----+
